STREAMS-1528 - Add documentation about new webhook testing endpoint (#54)
Axway/streams-open-docs: content/en/docs/subscribers/subscriber-webhook.md

| 200 Ok | Indicates that the subscription requested is valid and has been retrieved. |
| 404 Not found | Indicates that the requested URL or subscription requested does not exist. |

## Testing a webhook subscription

You can test a webhook subscription by making an HTTP Post request on the following endpoint:

`POST /subscribers/webhook/subscriptions/{subscriptionId}/test`

> The body could contain any JSON object and will be sent as is to the subscription identified.

Reviewer: ```suggestion
The request body can contain any JSON object and will be sent as is to the identified subscription.
```

### Test status codes

> Below the list of HTTP status codes that can be returned when trying to test a webhook subscription:

Reviewer: ```suggestion
Below the list of HTTP status codes that can be returned while testing a webhook subscription:
```

> Below the list of HTTP status codes that can be returned while testing a webhook subscription:

Reviewer 2: ```suggestion
The following HTTP status codes can be returned while testing a webhook subscription:
```

| Code | Comment |
|------|---------|
| 202 Accepted | Indicates that the payload has been successufuly sent to the subscription. |
| 400 Bad Request | Indicates that the provided data are invalid. |
| 404 Not found | Indicates that the requested URL does not exist. |

## Getting webhook subscriptions for a topic

> In order to get existing subscriptions, simply do the following GET request on your topic:

Reviewer 2: ```suggestion
To get existing subscriptions, do the following GET request on your topic:
```
